charts/trino/templates/_helpers.tpl

```diff
@@ -144,8 +144,8 @@ Create the secret name for the file-based authentication's password file
 Create the secret name for the group-provider file
 */}}
 {{- define "trino.groupsSecretName" -}}
-{{- if and .Values.auth .Values.auth.groupAuthSecret }}
-{{- .Values.auth.groupAuthSecret | trunc 63 | trimSuffix "-" }}
+{{- if and .Values.auth .Values.auth.groupsAuthSecret }}
```

Comment:
Can you also update the example in `values.yaml`, and regenerate the chart's README?

Reply:
makes sense! updated 👍